nx.json

```diff
@@ -53,6 +53,11 @@
       "dependsOn": [
         "^build"
       ]
+    },
+    "build:web-for-dashboard": {
+      "dependsOn": [
+        "^build"
+      ]
```

Comment:
When `nx build:web-for-dashboard @novu/web` command is running --> build all the web app dependencies first.